Quandl.Excel.UDF.Functions/UserDefinedFunctions/Timeseries.cs

```diff
@@ -35,7 +35,10 @@ public static class Timeseries
                 AllowReference = true)] string rawHeader = null,
             [ExcelArgument(Name = "dates",
                 Description = "(optional) Default: true - Whether the resulting data will include a dates column",
-                AllowReference = true)] string rawDateColumn = null
+                AllowReference = true)] string rawDateColumn = null,
+            [ExcelArgument(Name = "transpose",
+                Description = "(optional) Default: false - Transpose the resulting data matrix",
```

Comment:
could the wording be more explicitly to indicate it's vertically or horizontally display?